test/e2e/cypress/support/commands.ts

```diff
@@ -922,8 +922,9 @@ Cypress.Commands.add("logout", () => {
     const c = cookies.find(
       ({ name }) => name.indexOf("ory_kratos_session") > -1,
     )
-    expect(c).to.not.be.undefined
-    cy.clearCookie(c.name)
+    if (c) {
+      cy.clearCookie(c.name)
+    }
```

Comment:
The end goal is to remove the cookie. If it doesn't exist in the first place, this action should not fail, IMO.

Reply:
I am not sure I understand this, however it is fine by me to change the check as long as we have the assertion after login. Can you confirm that we assert the cookie after successful login?